pages/index.tsx

```diff
@@ -71,13 +71,13 @@ export default function Home() {
           <li>
             <div>
               Individuals who are unvaccinated or have not received their third booster dose{' '}
-              <span className='font-bold'>and</span>
+              <span className='font-bold'>and any of</span>
```

Comment:
> third booster dose and any of -> Have three or more chronic

I think this is how it reads, is that the intention?

Reply:
![image](https://user-images.githubusercontent.com/71518072/159381236-d379259c-71d8-491b-9c5a-29cd49f86f51.png)

Reply:
hmm - good call. that's what they said to update, but I'll double check.

Reply:
updated and requesting your re-approval.